ee/tasks/subscriptions/subscription_utils.py

```diff
@@ -41,10 +42,6 @@ def generate_assets(
     tasks = [exporter.export_asset.s(asset.id) for asset in assets]
     parallel_job = group(tasks).apply_async()
 
-    start_time = datetime.now()
-    while not parallel_job.ready():
-        sleep(1)
-        if datetime.now() > start_time + ASSET_GENERATION_MAX_TIMEOUT:
-            raise Exception("Timed out waiting for exports")
+    wait_for_parallel_celery_group(parallel_job, max_timeout=ASSET_GENERATION_MAX_TIMEOUT)
```

Comment:
This has nothing to do with the usage report?

Reply:
Yeah it's just that I took out the code for waiting for parallel tasks into a util function